labelspark/create_dataset.py

```diff
@@ -1,31 +1,134 @@
 #this code block is needed for backwards compatibility with older Spark versions
+from labelbox.schema.data_row_metadata import DataRowMetadataKind as metadata_type
 from pyspark import SparkContext
 from packaging import version
+import time
 sc = SparkContext.getOrCreate()
 if version.parse(sc.version) < version.parse("3.2.0"):
   import databricks.koalas as pd
   needs_koalas = True
 else:
   import pyspark.pandas as pd
-  needs_koalas = False
+  needs_koalas = False  
 
-# upload spark dataframe to Labelbox
-def create_dataset(client, spark_dataframe, iam_integration = 'DEFAULT', **kwargs):
-    # expects spark dataframe to have two columns: external_id, row_data
-    # external_id is the asset name ex: "photo.jpg"
-    # row_data is the URL to the asset
-    if needs_koalas:
-      spark_dataframe = spark_dataframe.to_koalas()
-    else:
-      spark_dataframe = spark_dataframe.to_pandas_on_spark()
-    dataSet_new = client.create_dataset(iam_integration = iam_integration, **kwargs)
+def create_dataset(client, spark_dataframe, dataset_name=str(time.now()), iam_integration='DEFAULT', metadata_index=False, **kwargs):
```

Comment:
Use of time.now() causes an error when library is packaged and invoked in Databricks. I think this can be solved by doing a:

from time import time

Here is the error for reference: 

module 'time' has no attribute 'now'

Reply:
I have discovered that the time module in Databricks doesn't have a "now" function (hilarious). The fix is 

from datetime import datetime

datetime.now()

Reply:
Updated